packages/lib/server/queries/teams/index.ts

```diff
@@ -260,6 +260,104 @@ export async function getTeamWithMembers(args: {
   };
 }
 
+export async function getMinimalTeam(args: {
+  id?: number;
+  slug?: string;
+  userId?: number;
+  orgSlug?: string | null;
+  /**
+   * If true, means that you are fetching an organization and not a team
+   */
+  isOrgView?: boolean;
+}) {
+  const { id, slug, userId, orgSlug, isOrgView } = args;
+
+  let lookupBy;
```

Comment:
How is TypeScript not complaining about  unexpected inferred `any`?

Reply:
Because the type is automatically derived by the below if/else. So, it is not any after line 284